### DISCUSSION TITLE
Why does the homebrew tap formula file not have an entry for Linux i386?

### DISCUSSION THREAD

**Discussion by TheZoraiz** [goreleaser/goreleaser#2382] [Q&A] [answered]
In my project, I'm generating binaries for Linux i386 platform but the `homebrew tap formula` file doesn't have its entry. At first I thought there was something wrong with my `.goreleaser.yaml` file, but after inspecting the code at `internal/pipe/brew/template.go`, I can see that i386 architecture isn't even included in the release.

Why is this the case? Does homebrew not support Linux i386 or something?

**Comment by caarlos0** [comment, marked as answer]
because we only coded arm and x86_64... if you think this should exists we can consider it a feature request

**Reply by TheZoraiz** [reply]
Ah, I see. Thanks for the clarifying.
